qt/aqt/main.py

```diff
@@ -1068,9 +1068,9 @@ def setupStyle(self) -> None:
         if is_lin:
             # On Linux, the check requires invoking an external binary,
             # which we don't want to be doing frequently
-            interval_secs = 300
+            interval_secs = 10
```

Comment:
300s seemed overkill. I don't know how much of a performance impact invoking the external binary has, but I didn't notice anything when I set it to `2` on my Gnome system. `5` or `10` would also suffice to achieve a relatively smooth transition between system themes.

Automatic theme adjustments are a very nice thing to have, especially if you switch themes regularly via shortcut or with a sunset timer.

Reply:
Yes, I was probably a little too conservative there :-) They appear to take a few ms at most, so once every 5-10s isn't going to contribute much to global warming/exhaust user's batteries.